FlexMeasures/flexmeasures
# 875 remove obsolete data classes from db

## Description

Remove obsolete tables from the DB if they are empty, otherwise give the user the choice to continue anyway.

## Look & Feel

run `flexmeasures db upgrade`

## How to test

If all the tables that will be removed are empty there will not be asked for confirmation. To test the check you can add a table name that has data to the table list and run db upgrade.

## Further Improvements

Potential improvements to be done in the same PR or follow up Issues/Discussions/PRs.

## Related Items

Mention if this PR closes an Issue or Project.

---
